### PR TITLE
Allow labels to contain emoji

### LINKED ISSUE #6062: Allow labels to have emojis
Github supports labels which contain emojis (see [here](https://github.blog/2018-02-22-label-improvements-emoji-descriptions-and-more/)).

I think it would be neat if Gitea supports that too :smile: :heavy_check_mark: :dancer: :poodle:

### PR DESCRIPTION
**Pull request by jolheiser**
Resolves #6062  

In my opinion they seem a little bulky...
Look at below comments for updated images.
issue list emoji
issue view emoji

### CODE REVIEW

File: options/locale/locale_es-ES.ini [hunk 1]
```diff
@@ -416,8 +416,8 @@ issues.create_label=Crear etiqueta
 issues.label_templates.title=Carga un conjunto predefinido de etiquetas
 issues.label_templates.helper=Seleccionar un conjunto de etiquetas
 issues.label_templates.fail_to_load_file=Error al cargar el archivo de plantilla de etiqueta '%s': %v
-issues.add_label_at=añadida la etiqueta <div class="ui label" style="color: %s\; background-color: %s">%s</div> %s
-issues.remove_label_at=eliminada la etiqueta <div class="ui label" style="color: %s\; background-color: %s">%s</div> %s
+issues.add_label_at=añadida la etiqueta <div class="ui label has-emoji" style="color: %s\; background-color: %s">%s</div> %s
```
Comment (techknowlogick) [inline options/locale/locale_es-ES.ini]: These translation files (the ones not en-US) will be overwritten by the CI when merged into master. The changes will have to be added to crowdin

Comment (jolheiser) [reply options/locale/locale_es-ES.ini]: RIP.
Okay, I'll back those changes out to clean up the PR and make the changes on crowdin.

**Comment by markusamshove** [comment]
I did the same find and replace to test this out locally, but you've beat me in creating a PR :-)

I've seen that in the labels overview the labels to have a label icon at the beginning, maybe this has a different css class and fits better?

**Comment by jolheiser** [comment]
Looks as though Semantic got excited about label images and made their height `!important`
semantic

As much as I hate using `!important` to fight other `!important` rules, it seems that until we re-design without Semantic this might be the best choice. I will add another style for these labels for now to correct the emoji size.

Compare these new images with the old ones in the original comment.
issue list emoji 2
issue view emoji 2

**Comment by silverwind** [comment]
Semantic is full of weird hacks like that :cry: 

Looking at your screenshots, I think the emojis are still a little bit too big and seem not exactly vertically centered in some cases. Maybe you could get them a bit closer to GitHub's styling:

````css
g-emoji {
    font-family: Apple Color Emoji,Segoe UI,Segoe UI Emoji,Segoe UI Symbol;
    font-size: 1.2em;
    font-weight: 400;
    line-height: 20px;
    vertical-align: middle;
}

g-emoji img {
    height: 1em;
    width: 1em;
}

.IssueLabel .g-emoji {
    display: inline-block;
    font-size: 1em;
    line-height: 1;
    position: relative;
    top: -.05em;
}

.IssueLabel--big .g-emoji {
    display: inline-block;
    margin-top: -1px;
}
````

**Comment by jolheiser** [comment]
Hmmm....I'm not sure I agree. This is at `1em` and I think it's a tad *too* small, some of the emoji are getting hard to recognize.  
Ultimately I'll go with what most people prefer, but as it stands I think I prefer the images above this one.  
The images above are at `1.5em` for reference, so there's technically `.5em` lee-way to work with. 😉 
issue view emoji 3

**Comment by silverwind** [comment]
I like them at `1em`, but wouldn't mind a bit bigger, maybe `1.2em`. Maybe also match the size of emojis in regular text (I imagine anything bigger than `1em` will introduce unwanted whitespace in the line).

**Comment by jolheiser** [comment]
I think `1.2em` looks fine.
issue view emoji 4

**Review by silverwind — APPROVED** [review]

**Review by sapk — APPROVED** [review]

**Review by adelowo — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -366,7 +366,7 @@ stylesheets-check: generate-stylesheets
 .PHONY: generate-stylesheets
 generate-stylesheets:
 	$(eval BROWSERS := "> 2%, last 2 firefox versions, last 2 safari versions")
-	node_modules/.bin/lessc --clean-css public/less/index.less public/css/index.css
+	node_modules\\.bin\\lessc --clean-css public/less/index.less public/css/index.css
```
Comment (lafriks) [inline Makefile]: Please revert this change

**Comment by lafriks** [comment]
Ouch... It was for 1.9.0... sry, already merged :)

**Comment by jolheiser** [comment]
Ugh, sorry about the Makefile. I've been trying to keep up with that and removing it before committing. 
Darn Windows being difficult.
